core/shared/src/test/scala/scalaz/zio/BaseCrossPlatformSpec.scala

```diff
@@ -46,4 +49,14 @@ abstract class BaseCrossPlatformSpec extends Specification with DefaultRuntime {
     unsafeRunToFuture(io.sandbox.mapError(FiberFailure(_))).map(p.success)
     p.future
   }
+
+  implicit class ZIOMustExpectable[R, E, A](zio: ZIO[R, E, A]) {
+
+    def must_===(other: => A)(implicit di: Diffable[A]): ZIO[R, E, MatchResult[A]] =
```

Comment:
WDYT about this one? Is it worth or should I inline `.map(a => a must_=== other)`?
Is using same name `must_===` OK or confusing?

Reply:
I think in the context of the library, it's great!